parser/jsonparser/jsonparser_test.go

```diff
@@ -0,0 +1,44 @@
+package jsonparser
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+func TestJsonParse(t *testing.T) {
+	parser := NewJsonParser()
+	line := `{"time_local":"2021-02-03T11:22:33+08:00","request_length":123,"request_method":"GET","request":"GET /order/2145 HTTP/1.1","body_bytes_sent":518,"status": 200,"request_time":0.544,"upstream_response_time":"0.543"}`
+
+	got, err := parser.ParseString(line)
+	if err != nil {
+		t.Error(err)
+	}
```

Comment:
Just as a side note, this module already uses the [testify package](https://github.com/stretchr/testify) for assertions in other test cases. These could make some of these assertions more concise.

```suggestion
	require.NoError(t, err)
```

Reply:
I shouldn't be careless here, thanks.